app/assets/i18n/_missing_translations_zh_HK.json

```diff
@@ -5,36 +5,36 @@
   ],
   "settingsTab": {
     "general": {
-      "showInContextMenu": "Show LocalSend in context menu"
+      "showInContextMenu": "喺檔案功能表嘅「傳送到」項目顯示 LocalSend"
     },
     "network": {
-      "discoveryTimeout": "Discovery Timeout"
+      "discoveryTimeout": "裝置搜尋逾時"
     }
   },
   "receiveHistoryPage": {
     "entryActions": {
-      "showInFolder": "Show in folder"
+      "showInFolder": "喺檔案瀏覽器顯示"
```

Comment:
I assume you mean “Show in _File Explorer_”

Reply:
I didn't want to mention "explorer" because this is only correct on Windows and not on macOS for example

Reply:
Ah, I mean file manager/broswer (the latter is what I translated into)